crates/doublets/src/handler.rs

```diff
@@ -15,38 +15,79 @@ impl From<bool> for Flow {
   }
 }
 
+impl<T> From<Option<T>> for Flow {
+  fn from(value: Option<T>) -> Self {
+    if value.is_some() { Flow::Continue } else { Flow::Break }
+  }
+}
```

Comment:
Is it useful? I mean to allow break with concreate values like `Flow::Break(value)`
This conversion is useless i mean